pkg/tools/sandbox/builder.go

```diff
@@ -43,40 +43,41 @@ import (
 )
 
 const (
-	defaultContextTimeout = time.Second * 15
-	// DefaultRegistryExpiryDuration is a default expiry duration for registry
-	DefaultRegistryExpiryDuration = 500 * time.Millisecond
+	defaultContextTimeout         = time.Second * 15
```

Comment:
Looks like I missed something
When we increased this?
It should be 1-5 sec

Reply:
Nope, it is an initial value from https://github.com/networkservicemesh/sdk/pull/460.

Reply:
Nope, it is an inital value from https://github.com/networkservicemesh/sdk/pull/460.

Reply:
Oops! Let's reduce this to 5s :)

Reply:
Reduced.